common/network_constants.cc

```diff
@@ -16,6 +16,8 @@ const char kBraveReferralsHeadersPath[] = "/promo/custom-headers";
 const char kBraveReferralsInitPath[] = "/promo/initialize/nonua";
 const char kBraveReferralsActivityPath[] = "/promo/activity";
 
+const char kBraveWebcompatReportEndpoint[] = "http://laptop-updates-pre.brave.com/1/webcompat";
```

Comment:
Is this the final URL?

Please make it HTTPS (the site already has a valid cert).

Reply:
No updates from @aekeus on that front so far unfortunately, but that is intended to be changed.

Reply:
Updated to HTTPS.